charts/datree-admission-webhook/values.yaml

```diff
@@ -61,11 +61,13 @@ affinity: {}
 tolerations: []
 scanJob:
   resources: {}
+  ttlSecondsAfterFinished: 100
   image:
```

Comment:
what bug does this fix?

Reply:
https://codelaborate.atlassian.net/jira/software/c/projects/DAT/boards/10?modal=detail&selectedIssue=DAT-4664